docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+       "    }\n",
+       "  })({\"config\": {\"view\": {\"continuousWidth\": 400, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-b19b5f45853c1a9ea92ff443230ae01b\"}, \"mark\": \"bar\", \"encoding\": {\"x\": {\"field\": \"month_str\", \"sort\": [\"Jan\", \"Feb\", \"Mar\", \"Apr\", \"May\", \"Jun\", \"Jul\", \"Aug\", \"Sep\", \"Oct\", \"Nov\", \"Dec\"], \"type\": \"ordinal\"}, \"y\": {\"field\": \"n_donations\", \"type\": \"quantitative\"}}, \"$schema\": \"https://vega.github.io/schema/vega-lite/v4.17.0.json\", \"datasets\": {\"data-b19b5f45853c1a9ea92ff443230ae01b\": [{\"month_int\": null, \"n_donations\": 1514, \"total_amount\": 250297, \"mean_amount\": 165.3216644649934, \"median_amount\": 100, \"month_str\": null}, {\"month_int\": 1.0, \"n_donations\": 348979, \"total_amount\": 174837854, \"mean_amount\": 500.9982090612902, \"median_amount\": 121, \"month_str\": \"Jan\"}, {\"month_int\": 2.0, \"n_donations\": 581646, \"total_amount\": 255997655, \"mean_amount\": 440.126219384299, \"median_amount\": 100, \"month_str\": \"Feb\"}, {\"month_int\": 3.0, \"n_donations\": 1042577, \"total_amount\": 430906797, \"mean_amount\": 413.3093258339672, \"median_amount\": 81, \"month_str\": \"Mar\"}, {\"month_int\": 4.0, \"n_donations\": 1088244, \"total_amount\": 299252692, \"mean_amount\": 274.98676032213365, \"median_amount\": 50, \"month_str\": \"Apr\"}, {\"month_int\": 5.0, \"n_donations\": 1374248, \"total_amount\": 387317202, \"mean_amount\": 281.83937833637015, \"median_amount\": 48, \"month_str\": \"May\"}, {\"month_int\": 6.0, \"n_donations\": 1667285, \"total_amount\": 465305247, \"mean_amount\": 279.07960966481437, \"median_amount\": 44, \"month_str\": \"Jun\"}, {\"month_int\": 7.0, \"n_donations\": 1607053, \"total_amount\": 320528605, \"mean_amount\": 199.45117242555162, \"median_amount\": 35, \"month_str\": \"Jul\"}, {\"month_int\": 8.0, \"n_donations\": 2023466, \"total_amount\": 473544182, \"mean_amount\": 234.02626088108227, \"median_amount\": 35, \"month_str\": \"Aug\"}, {\"month_int\": 9.0, \"n_donations\": 2583847, \"total_amount\": 697888624, \"mean_amount\": 270.0967294116099, \"median_amount\": 38, \"month_str\": \"Sep\"}, {\"month_int\": 10.0, \"n_donations\": 3686024, \"total_amount\": 850820707, \"mean_amount\": 230.82343115508743, \"median_amount\": 29, \"month_str\": \"Oct\"}, {\"month_int\": 11.0, \"n_donations\": 2545616, \"total_amount\": 285143995, \"mean_amount\": 112.01375030640914, \"median_amount\": 25, \"month_str\": \"Nov\"}, {\"month_int\": 12.0, \"n_donations\": 2119311, \"total_amount\": 283081648, \"mean_amount\": 133.57249030463203, \"median_amount\": 25, \"month_str\": \"Dec\"}]}}, {\"mode\": \"vega-lite\"});\n",
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.Chart(...)"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "months_in_order = list(month_map.values())\n",
```

Comment:
Not sure if it's just me, but the sorted table isn't rendering.

Reply:
See my longer response below, still figuring this out.